File: patch_types.py
with open('src/types.ts', 'r') as f:
    content = f.read()

content = content.replace('| "jhun_brush"', '| "jhun_brush"\n    | "pen_story"')

with open('src/types.ts', 'w') as f:
    f.write(content)

File: patch-types.js
import fs from 'fs';
let code = fs.readFileSync('src/types.ts', 'utf8');

code = code.replace(
  '| "layered_outline"', 
  '| "layered_outline"\n    | "calli"'
);
code = code.replace(
  '| "brush_stroke"', 
  '| "brush_stroke"\n    | "calli"'
);

fs.writeFileSync('src/types.ts', code);
console.log("Types patched");
